experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -108,7 +108,7 @@ export class SchemaSummarizer implements Summarizable {
 		let schemaBuffer: ArrayBufferLike;
 		if (hasBlob) {
 			const handleBuffer = await services.readBlob(schemaBlobKey);
-			const handleString = bufferToString(handleBuffer, "utf-8");
+			const handleString = bufferToString(handleBuffer, "utf8");
```

Comment:
Because we consider 'client-utils' to be internal to the client release group, its safe/non-breaking to require "utf8".